source/trackers.json

```diff
@@ -1627,8 +1627,11 @@
         "adservice.google.pl": "google_marketing",
         "adservice.google.ru": "google_marketing",
         "adservice.google.vg": "google_marketing",
+        "adtrafficquality.google": "google_marketing",
         "dai.google.com": "google_marketing",
         "doubleclickbygoogle.com": "google_marketing",
+        "ep1.adtrafficquality.google": "google_marketing",
+        "ep2.adtrafficquality.google": "google_marketing",
```

Comment:
subdomains of adtrafficquality.google look redundant. Or not?

Reply:
Seems subdomains will match